macosx.sh

```diff
@@ -0,0 +1,135 @@
+              -DCMAKE_INSTALL_PREFIX=${install_dir}/moab
+    make
+    make install
+    
+    echo "if [ -n \"\${LD_LIBRARY_PATH-}\" ]" >> ~/.bashrc
+    echo "then" >> ~/.bashrc >> ~/.bashrc
+    echo "  export LD_LIBRARY_PATH=${install_dir}/moab/lib:\$LD_LIBRARY_PATH" >> ~/.bashrc
+    echo "else" >> ~/.bashrc
+    echo "  export LD_LIBRARY_PATH=${install_dir}/moab/lib" >> ~/.bashrc
+    echo "fi" >> ~/.bashrc
+
+    PYTHON_VERSION=$(python -c 'import sys; print(sys.version.split('')[0][0:3])')
+    echo "if [ -n \"\${PYTHONPATH-}\" ]" >> ~/.bashrc
+    echo "then" >> ~/.bashrc >> ~/.bashrc
+    echo "  export PYTHONPATH=$install_dir/moab/lib/python${PYTHON_VERSION}/site-packages:\$PYTHONPATH" >> ~/.bashrc
+    echo "else" >> ~/.bashrc
+    echo "  export PYTHONPATH=$install_dir/moab/lib/python${PYTHON_VERSION}/site-packages" >> ~/.bashrc
+    echo "fi" >> ~/.bashrc
+    source ~/.bashrc
+}
+
+function build_dagmc {
+
+    # Install DAGMC
+    cd ${install_dir}
+    check_repo dagmc
+    mkdir -p dagmc
+    cd dagmc
+    git clone https://github.com/svalinn/DAGMC.git dagmc-repo
+    cd dagmc-repo
+    git checkout develop
+    mkdir build
+    cd build
+    cmake .. -DMOAB_DIR=${install_dir}/moab \
+             -DBUILD_STATIC_LIBS=OFF \
+             -DCMAKE_INSTALL_PREFIX=${install_dir}/dagmc
+    make
+    make install
+}
+
+function install_pyne {
+
+    # Install PyNE
+    cd $install_dir
+    check_repo pyne
+    git clone https://github.com/pyne/pyne.git
+    cd pyne
+    if [ $1 == 'stable' ] ; then
+        TAG=$(git describe --abbrev=0 --tags)
+        git checkout tags/`echo $TAG` -b `echo $TAG`
+    fi
+    python setup.py install --user -- -DMOAB_LIBRARY=$install_dir/moab/lib -DMOAB_INCLUDE_DIR=$install_dir/moab/include
+    echo "export PATH=$HOME/.local/bin:\$PATH" >> ~/.bashrc
+    echo "export LD_LIBRARY_PATH=$HOME/.local/lib:\$LD_LIBRARY_PATH" >> ~/.bashrc
+    echo "alias build_pyne='python setup.py install --user -- -DMOAB_LIBRARY=$install_dir/moab/lib -DMOAB_INCLUDE_DIR=$install_dir/moab/include'" >> ~/.bashrc
+    PYTHON_VERSION=$(python -c 'import sys; print(sys.version.split('')[0][0:3])')
+    echo "if [ -n \"\${PYTHONPATH-}\" ]" >> ~/.bashrc
+    echo "then" >> ~/.bashrc >> ~/.bashrc
+    echo "  export PYTHONPATH=~/.local/lib/python${PYTHON_VERSION}/site-packages:\$PYTHONPATH" >> ~/.bashrc
+    echo "else" >> ~/.bashrc
+    echo "  export PYTHONPATH=~/.local/lib/python${PYTHON_VERSION}/site-packages" >> ~/.bashrc
+    echo "fi" >> ~/.bashrc
+}
+
+function run_nuc_data_make {
+
+    cd
+    source ~/.bashrc
+    # Generate nuclear data file
+    nuc_data_make
+
+}
+
+function test_pyne {
+    
+    source ~/.bashrc
+    cd $install_dir/pyne
+    cd tests
+
+    nosetests .
+}
+
+
+# system update
+eval brew update
+eval brew install $brew_package_list
+export PATH="$HOME/.local/bin:$PATH"
+eval sudo pip3 install $pip_package_list
+
+install_dir=$HOME/opt
+mkdir -p $install_dir
+
+build_moab
+
```

Comment:
```suggestion

build_dagmc
```

Reply:
I think that's the last thing missing.

@gonuke once this is added, I thing this should be ready for final review